silk/tools/otns_manager.py

```diff
@@ -1056,8 +1056,12 @@ def unsubscribe_from_all_nodes(self):
       subscriber.unsubscribe()
     self.otns_monitor_map.clear()
 
-  def update_layout(self):
+  def update_layout(self, use_two_layer=False):
```

Comment:
Did you compare the visualization for running silk test cases with and without using two layers which can decide the selection of the default.

Reply:
I tested single cluster and feel that keeping them stationary looks better. Couldn't test running more clusters yet. I feel that this can be the default option, and the two layer code is only left there if we want to use it to debug something. But if you feel that we should either provide an argument option or remove it completely I can do that too.

Reply:
Let's keep as is and decide if it's needed to have an argument option after running more tests.